core/src/test/java/org/apache/iceberg/variants/TestPrimitiveWrapper.java

```diff
@@ -57,7 +59,13 @@ public class TestPrimitiveWrapper {
         Variants.of(new BigDecimal("9876543210.123456789")), // decimal16
         Variants.of(new BigDecimal("-9876543210.123456789")), // decimal16
         Variants.of(ByteBuffer.wrap(new byte[] {0x0a, 0x0b, 0x0c, 0x0d})),
-        Variants.of("iceberg"),
+        Variants.of("iceberg"), // short string below 255 bytes to test value offset size of 1
```

Comment:
Another minor note here, We don't actually check that we did a "short stream" serde here, we probably want an explicit test. For example, we do not know if these tests are hitting the long or short string paths.